packages/connectivity/src/scp-cf/destination/destination-from-registration.ts

```diff
@@ -70,9 +70,21 @@ export type DestinationWithName = Destination & { name: string };
 export function searchRegisteredDestination(
   options: DestinationFetchOptions
 ): Destination | null {
+  let decodedJwt;
+  // An error could be thrown if no jwt and no xsuaa service exist.
+  try {
+    decodedJwt = decodedJwtOrZid(options);
+  } catch (e) {
+    logger.debug(
+      'Could not retrieve destination from registered destination. Failed to decode jwt or create a dummy jwt with `zid` property.'
```

Comment:
```suggestion
      'Failed to retrieve registered destination, because it was neither possible to decode jwt nor create a dummy jwt with `zid` property.'
```
[pp] The original message seems confusing to me. I tried to rephrase (hopefully keeping the correct meaning).

Reply:
Thanks! I think your version is more clear. 👍